Source/Core/Core/IPC_HLE/WII_IPC_HLE_Device_sdio_slot0.cpp

```diff
@@ -63,19 +63,28 @@ void CWII_IPC_HLE_Device_sdio_slot0::OpenInternal()
 {
 	const std::string filename = File::GetUserPath(D_WIIROOT_IDX) + "/sd.raw";
 	m_Card.Open(filename, "r+b");
+	if (m_Card)
+		return;
+	
+	WARN_LOG(WII_IPC_SD, "Failed to open SD Card image, trying read-only access...");
+	m_Card.Open(filename, "rb");
+	if (m_Card)
+	{
+		SConfig::GetInstance().bEnableMemcardSdWriting = false;
```

Comment:
Actually, there may be no point in opening the file in r+b mode anyway, since Dolphin will attempt to open the file again with r+b access when trying to write to it.

https://github.com/dolphin-emu/dolphin/blob/master/Source/Core/Core/HW/GCMemcardRaw.cpp#L93

Maybe dolphin should default to readonly here?